helm/botkube/values.yaml

```diff
@@ -250,6 +250,21 @@ config:
     # Set false to disable upgrade notification
     upgradeNotifier: true
 
+service:
+  name: metrics
+  port: 2112
+  targetPort: 2112  
+            
+serviceMonitor:
+  ## If true, a ServiceMonitor CRD is created for a botkube
+  ## https://github.com/infracloudio/botkube
```

Comment:
We should write `CR` instead of `CRD` here. 
Also how about adding link to Pormetheus Operator GitHub instead of BotKube repository?
May be this? https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#servicemonitor